metricbeat/module/system/process/helper.go

```diff
@@ -373,6 +383,11 @@ func (procStats *ProcStats) GetProcStats() ([]common.MapStr, error) {
 	}
 	procStats.ProcsMap = newProcs
 
+	var summaries *processSummaries
+	if procStats.Summaries {
+		summaries = procStats.getSummaries(processes)
```

Comment:
Ah, you are right, it should be unfiltered, I didn't realize that the regexps are already applied at that point.